static/app/views/ddm/metricsExplorer.tsx

```diff
@@ -219,12 +225,19 @@ function MetricSearchBar({tags, mri, disabled, onChange}: MetricSearchBarProps)
     async tag => {
       const tagsValues = await api.requestPromise(
         `/organizations/${org.slug}/metrics/tags/${tag.key}/`,
-        {query: {useCase: getUseCaseFromMri(mri)}}
+        {
+          query: {
+            // TODO(ddm): OrganizationMetricsTagDetailsEndpoint does not return values when metric is specified
```

Comment:
Is this a bug or known behavior that we want to change?

Reply:
Could it be that it's because it is accepting a list (https://github.com/getsentry/sentry/blob/main/src/sentry/api/endpoints/organization_metrics.py#L129-L129) and you are passing a single value?

Reply:
It's either a bug or it was never intended to accept metric (but the backend code says otherwise). Passing a list unfortunately does not help. I created a ticket for us to take a look 👍